Makefile

```diff
@@ -19,25 +29,25 @@ DOCKERS=docker_edgex_ui_go
 
 VERSION=$(shell cat ./VERSION 2>/dev/null || echo 0.0.0)
 
-GOFLAGS=-ldflags "-X github.com/edgexfoundry/edgex-ui-go.Version=$(VERSION)"
+GOFLAGS=-ldflags "-X github.com/edgexfoundry/edgex-ui-go.Version=$(VERSION)" -trimpath -mod=readonly
```

Comment:
@badboy-huaqiao Maybe the make build error is caused by `-mod=readonly`, which prevents `go mod tidy`? Please see [logs from the snap testing build](https://github.com/edgexfoundry/edgex-ui-go/actions/runs/3152765332/jobs/5128401564#step:2:1688)

Reply:
great 👍, I try again

Reply:
Did you forget to run `go mod tidy` locally before committing?

Reply:
I have fully tested it on my local side,  do you mean running `go mod tidy`  again to update  `go.sum` file?

Reply:
yeah, I run the `go mod tidy`  again, and it indeed updates the go.sum file again on my local side,  let me try again.

Reply:
still not work, I saw that there is  a warning on my local side when running `make test-go`, it shows 

> An attribution for github.com/edgexfoundry/go-mod-bootstrap/v2 is missing from Attribution.txt, please add
will it cause failing? but that didn't  show it's an error on my local side.